src/coreclr/utilcode/log.cpp

```diff
@@ -185,11 +181,14 @@ VOID InitializeLogging()
     if (bLoggingInitialized)
         return;
 
-    HANDLE mutex = CreateMutex(NULL, FALSE, NULL);
-    _ASSERTE(mutex != 0);
-    if (InterlockedCompareExchangeT(&LogFileMutex, mutex, 0) != 0)
+    minipal_mutex* mutex = new minipal_mutex;
```

Comment:
Is allocation needed at all here? Can we replace this with a static and a pointer?